notebooks/building_production_ml_systems/labs/1_training_at_scale_vertex.ipynb

```diff
@@ -16,7 +16,7 @@
     "\n",
```

Comment:
<p>Change the data path (see solution notebook).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>